src/main/java/org/spongepowered/api/item/inventory/InventoryProperties.java

```diff
@@ -53,20 +57,23 @@
 
     /**
      * Represents the capacity of a inventory.
+     * TODO not present in impl | do we need this for query? {@link Inventory#capacity()} exists
+     *
      */
     public static final Property<Integer> CAPACITY =
             DummyObjectProvider.createFor(Property.class, "CAPACITY");
 
     /**
      * Represents the dimension of a inventory.
+     * TODO not present in impl | do we need this for query? {@link GridInventory#getRows()} and {@link GridInventory#getColumns()} exists
```

Comment:
If all Inventories effectively have a dimension, even if 1-dimensional, then I would say we still need this as someone may not care about if it is a grid and only cares about the particular dimensions of the inventory.

Reply:
Dimension for grids. Currently used to create grids for custom inventories, which is done differently with the new builders